jsonschema/src/error.rs

```diff
@@ -143,6 +142,13 @@ pub enum ValidationErrorKind {
     UniqueItems,
     /// Reference contains unknown scheme.
     UnknownReferenceScheme { scheme: String },
+    /// Error during schema ref resolution.
```

Comment:
Should it be behind `cfg` as it was with now removed `Reqwest` variant?

Reply:
I don't think so, as even if `resolve-http` and `resolve-file` are not selected, the user of the library can still provide their own resolver.

If it was put behind `cfg`, `SchemaResolver` and all related items should be behind `cfg` as well.

Also, in the previous version, a panic happened if the feature was not enabled, but now it's an error instead. While technically not a validation error, I'd still generally prefer this behaviour where possible as accidental panics can bring down entire applications.

Reply:
Aha, makes sense :)